### PR TITLE
chore: flappy bird ui fixes

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2024-01-19 at 11 06 12@2x

after:

CleanShot 2024-01-19 at 11 08 53@2x

### CODE REVIEW

File: packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx [hunk 1]
```diff
@@ -552,8 +552,8 @@ const RedirectToInstantMeetingModal = ({ expiryTime }: { expiryTime?: Date }) =>
               */}
 
               <p className="font-medium">{t("please_do_not_close_this_tab")}</p>
-              <div className="h-[450px]">
-                <iframe className="mx-auto h-full" src="https://cal.games/" />
+              <div className="mt-4 h-[450px]">
```
Comment (PeerRich) [inline packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx:555]:
```suggestion
              <div className="mt-4 h-[414px]">
```

File: packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx [hunk 2]
```diff
@@ -552,8 +552,8 @@ const RedirectToInstantMeetingModal = ({ expiryTime }: { expiryTime?: Date }) =>
               */}
 
               <p className="font-medium">{t("please_do_not_close_this_tab")}</p>
-              <div className="h-[450px]">
-                <iframe className="mx-auto h-full" src="https://cal.games/" />
+              <div className="mt-4 h-[414px]">
```
Comment (Udit-takkar) [inline packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx:555]: I already added margin top here on my PR https://github.com/calcom/cal.com/pull/13258/

Comment (Udit-takkar) [reply packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx:555]: can you push these changes on my PR feat: instant meeting improvements #13258?

Comment (PeerRich) [inline packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx:555]:
```suggestion
              <div className="h-[414px]">
```

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**